README.md

```diff
@@ -1,96 +1,127 @@
+
+### Databricks Dispatch Configuration
+If you are using a Databricks destination with this package you will need to add the below (or a variation of the below) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+```yml
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
+```
 
+## Step 2: Install the package
+Include the following zendesk_source package version in your `packages.yml` file **only if you are NOT also installing the [Zendesk transformation package](https://github.com/fivetran/dbt_zendesk)**. The transform package has a dependency on this source package.
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 ```yaml
 packages:
   - package: fivetran/zendesk_source
-    version: [">=0.6.0", "<0.7.0"]
+    version: [">=0.7.0", "<0.8.0"]
 ```
-
-## Configuration
-By default, this package looks for your Zendesk data in the `zendesk` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Zendesk Support data is, add the following configuration to your `dbt_project.yml` file:
+## Step 3: Define database and schema variables
+By default, this package runs using your target database and the `zendesk` schema. If this is not where your Zendesk data is (for example, if your zendesk schema is named `zendesk_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-    zendesk_schema: your_schema_name
-    zendesk_database: your_database_name
+    zendesk_database: your_destination_name
+    zendesk_schema: your_schema_name 
+```
+## Step 4: Disable models for non-existent sources
+This package takes into consideration that not every Zendesk account utilizes the `schedule`, `domain_name`, `user_tag`, `organization_tag`, or `ticket_form_history` features, and allows you to disable the corresponding functionality. By default, all variables' values are assumed to be `true`. Add variables for only the tables you want to disable:
+```yml
+vars:
+    using_schedules:            False         #Disable if you are not using schedules
+    using_domain_names:         False         #Disable if you are not using domain names
+    using_user_tags:            False         #Disable if you are not using user tags
+    using_ticket_form_history:  False         #Disable if you are not using ticket form history
+    using_organization_tags:    False         #Disable if you are not using organization tags
 ```
 
-### Adding Passthrough Columns
+## Step 5: Adding passthrough columns
```

Comment:
I feel this should be included in the optional steps. Thoughts?

Also, it looks like we have two `Step 5:` sections.

Reply:
oh yeah definitely, i'll move it there